### DISCUSSION TITLE
My filters/Strict blocking not getting applied?

### DISCUSSION THREAD

**Discussion by tod222** [uBlockOrigin/uBlock-issues#1916] [Q&A] [answered]
While investigating an unchanged My filters config with some rules that stopped working, I noticed that the first line which enables strict blocking on all URLs using `|*|$all` has _also_ stopped working.

Before I create an issue I figured I'd ask if anything has changed, since the last time the strict blocking enabled by this config broke was due to a syntax change (#687).

Firefox 95.0.1
uBlock Origin 1.40.2

**Comment by gorhill** [comment, marked as answer]
Yes, I confirm there is an issue, thanks for reporting.

Meanwhile, the workaround is to simply use a single wildcard, there is no need for the anchors:

    *$all

**Comment by gorhill** [comment]
Fixed with <https://github.com/gorhill/uBlock/commit/6836d2b9ca62b1532ce1d98d70df69ee08efffab>.
